apps/api/src/mailer/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "../../tsconfig.json",
+  "include": [
+    "src/**/*.ts",
+    "index.ts",
+    "src/**/*.json",
+    "../../../../libs/interface/mailer/index.ts"
+  ],
```

Comment:
I don't think that we need all of these files. Check other examples to fix here.